src/frontend/pages/login/login.js

```diff
@@ -7,26 +7,39 @@ import axios from "axios";
 function Login() {
   const navigate = useNavigate();
   const { authState, authDispatch } = useAuth();
-  const { user } = authState;
+  const { user, error } = authState;
 
   const loginHandler = async (e) => {
     e.preventDefault();
     try {
-      const response = await axios.post("/api/auth/login", {
+      const response = await axios.post(`/api/auth/login`, {
         email: user.email,
         password: user.password,
       });
       localStorage.setItem("token", response.data.encodedToken);
       authDispatch({
-        authType: "TOKEN",
-        authPayload: response.data.encodedToken,
+        type: "TOKEN",
+        payload: response.data.encodedToken,
       });
       navigate("/");
-    } catch (error) {
-      authDispatch({ authType: "ERROR", authPayload: true });
-      console.log(`Error: ${error}`);
+    } catch (err) {
+      authDispatch({
+        type: "ERROR",
+        payload:
+          "Wrong Email or Pssword, signup if you don't have account with us",
+      });
+      console.log(err);
```

Comment:
Really liked the idea of having different context for path and passing error messages as payload